### PR TITLE
fix: lifecycle review pairing

### PR DESCRIPTION
**Pull request by tgriesser**
Fixing various lifecycle bugs w/ @brian-mann & @ryanthemanuel 
- clears scaffold state when switching projects
- does not show loading browsers interim screen when scaffolding

### CODE REVIEW

File: packages/launchpad/cypress/e2e/global-mode-onboarding.cy.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    cy.openGlobalMode()
+    cy.addProject('pristine')
+    cy.visitLaunchpad()
+  })
+
+  it('scaffolds a project, go back to global screen, and state is cleared', () => {
+    cy.get('[data-cy=project-card]').click()
+    cy.get('[data-cy-testingtype="e2e"]').click()
+    cy.get('h1').should('contain', 'Configuration Files')
+    cy.get('[data-cy="global-mode-link"]').click()
```
Comment (emilyrohrbough) [inline packages/launchpad/cypress/e2e/global-mode-onboarding.cy.ts:13]:
```suggestion
    cy.findByRole('link', { name: 'Project' }).click()
```
Using the above should remove the need to add data-cy on this anchor tag

File: packages/launchpad/src/Main.vue [hunk 2]
```diff
@@ -67,7 +67,8 @@
           v-else-if="currentProject.currentTestingType === 'component' && !currentProject.isCTConfigured"
           :gql="query.data.value"
         />
-        <OpenBrowser v-else />
+        <OpenBrowser v-else-if="currentProject.browsers?.length" />
+        <Spinner v-else />
```
Comment (emilyrohrbough) [inline packages/launchpad/src/Main.vue:71]: nice! Was seeing this issue where the browsers would display when it should be a loading page

File: packages/launchpad/cypress/e2e/global-mode-onboarding.cy.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
+describe('Global Mode: Onboarding', () => {
```
[the remark this answers is not in the document]
Comment (tgriesser) [reply packages/launchpad/cypress/e2e/global-mode-onboarding.cy.ts]: Good idea, moved it there

File: packages/launchpad/src/Main.vue (hunk 2 above)
Comment (tgriesser) [reply packages/launchpad/src/Main.vue:71]: Yep - we went through and I think fixed a number of the edge cases causing this

File: packages/launchpad/cypress/e2e/global-mode-onboarding.cy.ts (hunk 1 above)
Comment (brian-mann) [reply packages/launchpad/cypress/e2e/global-mode-onboarding.cy.ts]: how does `{ name: 'Project' }` correspond to the `global-mode-link` data-cy attribute?

Comment (brian-mann) [reply packages/launchpad/cypress/e2e/global-mode-onboarding.cy.ts]: oh you're saying it corresponds to the link element with role attribute `<a role='Project' >...</a>`

**Comment by tgriesser** [comment]
Closing this one out, the issues that this one was fixing have been merged in other PRs

**Pull request closed without merging**